pkg/diff/chunk.go

```diff
@@ -162,43 +141,48 @@ func (c *ChunkRange) toString(collation string) (string, []string) {
 	}
 
 	return fmt.Sprintf("(%s) AND (%s)", strings.Join(lowerCondition, " OR "), strings.Join(upperCondition, " OR ")), append(lowerArgs, upperArgs...)
-
 }
 
-func (c *ChunkRange) update(column, lower, lowerSymbol, upper, upperSymbol string) {
-	newBound := &Bound{
-		Column:      column,
-		Lower:       lower,
-		LowerSymbol: lowerSymbol,
-		Upper:       upper,
-		UpperSymbol: upperSymbol,
-	}
-
+func (c *ChunkRange) update(column, lower, upper string) {
 	for i, b := range c.Bounds {
 		if b.Column == column {
 			// update the bound
-			c.Bounds[i] = newBound
+			if len(lower) > 0 {
+				c.Bounds[i].Lower = lower
```

Comment:
in fact it can't distinguish the lower/upper is "" or don't have lower/upper, I add `HasLower` and `HasUpper` in `Bound` in  https://github.com/pingcap/tidb-tools/pull/256/commits/040a4a27a08bb9494dba9c2924775a645b9bedc6